test/test_pmm.py

```diff
@@ -620,6 +679,18 @@ def test_inventory_skew_multiple_orders_status(self):
         self.assertEqual("50.0%", status_df.iloc[4, 1])
         self.assertEqual("150.0%", status_df.iloc[4, 2])
 
+    def test_order_book_asset_del(self):
+        strategy = self.one_level_strategy
+        strategy.asset_price_delegate = self.order_book_asset_del
+        self.clock.add_iterator(strategy)
+        self.clock.backtest_til(self.start_timestamp + 1)
+
+        bid = self.order_book_asset_del.get_price_by_type(PriceType.BestBid)
+        ask = self.order_book_asset_del.get_price_by_type(PriceType.BestAsk)
+        mid_price = self.order_book_asset_del.get_price_by_type(PriceType.MidPrice)
+
+        self.assertEqual((bid + ask) / 2, mid_price)
+
```

Comment:
@petioptrv Can you please add a test on `order_book_asset_del.get_price_by_type(PriceType.LastTrade)` here? 
You can simulate trades on the underlying order book, similar to how you did it for `test_basic_one_level_price_type`.
Thanks

Reply:
@Nullably I see, test case added

Reply:
@Nullably, test case added